cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -308,6 +315,20 @@ protected Future<Void> modelWarnings(KafkaStatus kafkaStatus) {
         return Future.succeededFuture();
     }
 
+    /**
+     * Initialize the TrustStore and KeyStore suppliers to be used by admin clients during reconciliation
+     *
+     * @return Completes when the TrustStore and KeyStore suppliers have been created
+     */
+    protected Future<Void> initAdminClientKeyStoreTrustStore() {
+        return ReconcilerUtils.clientSecrets(reconciliation, secretOperator)
```

Comment:
I wonder if we should push knowledge of the `Pem...` down into the ReconcilerUtils, so we have a `Future<PemKeyStoreSupplier> ReconcilerUtils.clientIdentity(Reconciliation reconciliation, SecretOperator secretOperator)` and likewise for the trust.

Reply:
Yeah I think that makes sense